.github/workflows/website-links.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         uses: lycheeverse/lychee-action@885c65f3dc543b57c898c8099f4e08c8afd178a2 # v2.6.1
         with:
           fail: false
-          args: --verbose --no-progress --exclude-all-private --max-concurrency 1 --retry-wait-time 60 --base https://firezone.dev .
+          args: --verbose --no-progress --exclude-all-private --max-concurrency 1 --retry-wait-time 60 --base https://firezone.dev --accept 100..=103,200..=299,429 .
```

Comment:
nitpick: it would be good to add a `FIXME` to remove accepting 429's when the condition changes, so that we won't forget about it (and e.g. IDE can help us)

Reply:
I am subscribed to the PR and will fix it once that lands!